server/Dockerfile

```diff
@@ -1,15 +1,32 @@
-FROM node:16-alpine3.14 as core
-
-ARG DEBIAN_FRONTEND=noninteractive
+# Build stage
+FROM node:16-alpine3.14 as builder
 
 WORKDIR /usr/src/app
 
 COPY package.json package-lock.json ./
 
-RUN apk add --update-cache build-base python3 libheif vips-dev vips ffmpeg
-
-RUN npm ci
+RUN apk add --update-cache build-base python3 libheif vips-dev \
+    && npm ci
```

Comment:
I would like to split the `npm ci` on its own line to distinguish the dependencies for the container environment vs the Node's dependencies installation.